### DISCUSSION TITLE
Welcome to ruby-vips Discussions!

### DISCUSSION THREAD

**Discussion by jcupitt** [libvips/ruby-vips#265] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Nakilon** [comment]
Interesting. There is a category tab like labels in Issues, but there is also upvoting like in Discourse.

**Reply by Nakilon** [reply]
And comments are in a tree form, but unlike comments on SO there is `Markdown` in them **I guess** so it's more like on Reddit.
